### DISCUSSION TITLE
Livewire components within blocks

### DISCUSSION THREAD

**Discussion by edjeavons** [jeffreyvr/dropblockeditor#37] [Q&A]
I've come across an issue when a block contains a Livewire component. It renders fine but it's not possible to open block's settings in the editor. Is there a way around this issue?

This seems to happen with any Livewire component that I use and the following video shows the issue in action. Essentially it doesn't respond to clicks on the block.

https://github.com/user-attachments/assets/4fdecb10-ea4f-4dc9-9f23-f25eb311b747

By the way this is a great package, thank you! I'm using it in a Filament project and will share the process in #36 once we've perfected a few more things.

**Comment by JacquesvanWyk** [comment]
@edjeavons I am also interested in how you use this with filament.
